python/chronos/src/bigdl/chronos/autots/model/auto_prophet.py

```diff
@@ -119,8 +119,9 @@ def fit(self,
                https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html#timeseries-offset-aliasesDefaulted
                to None, where an unreliable frequency will be infer implicitly.
         :param metric_threshold: a trial will be terminated when metric threshold is met
-        :param n_sampling: Number of times to sample from the search_space. Defaults to 50.
-               If hp.grid_search is in search_space, the grid will be repeated n_sampling of times.
+        :param n_sampling: Number of trials to evaluate in total. Defaults to 1.
+               If hp.grid_search is in search_space, the grid will be run n_sampling of trials
+               and round up n_sampling according to hp.grid_search.
```

Comment:
you need to change the n_sampling value here in fit and also in base automodel and auto_arima. they don't rely on autotsestimator